customer/src/main/java/no/unit/nva/customer/get/FetchChannelClaimHandler.java

```diff
@@ -39,7 +39,9 @@ protected void validateRequest(Void unused, RequestInfo requestInfo, Context con
     }
 
     private static void userIsAuthorized(RequestInfo requestInfo) throws UnauthorizedException {
-        requestInfo.getCurrentCustomer();
+        if (!requestInfo.clientIsInternalBackend()) {
+            requestInfo.getCurrentCustomer();
+        }
```

Comment:
Backend client needs to retrieve channel ownership when validating update and publishing.